ui_files/config.ui

```diff
@@ -58,6 +58,22 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="group_deck_import">
+         <property name="title">
+          <string>Import</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_5">
+          <item>
+           <widget class="QCheckBox" name="cb_ignore_move_cards">
+            <property name="text">
+             <string>Do Not Move Existing Cards</string>
```

Comment:
I think this warrant a more detailed explanation, potentially with a reference to a github issue

Reply:
Done :+1: give it a look